tools/c7n_azure/tests/test_tags.py

```diff
@@ -24,6 +24,37 @@ class TagsTest(BaseTest):
 
     existing_tags = {'tag1': 'value1', 'tag2': 'value2'}
 
+    def test_tag_schema_validate(self):
+        with self.sign_out_patch():
```

Comment:
Can you please move it to `test_actions_tag.py` (test_schema_validate function)? (`test_tags.py` now has TagHelper tests, all action-specific tests have their own files)

Reply:
I did not even see that test file. Will move.